backend/.env.test

```
@@ -2,3 +2,4 @@ DISCORD_TOKEN=""
 GITHUB_TOKEN=""
 OPENAI_API_KEY=""
 POSTGRES_URL=postgresql://postgres:postgres@localhost:5432/postgres
+EMBEDDINGS_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMBEDDINGS_ENABLED key should go before the GITHUB_TOKEN key

(UnorderedKey)